airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -398,9 +407,16 @@ public PostgresType getDatabaseFieldType(final JsonNode field) {
         // It should not be converted to base64 binary string. So it is represented as JDBC VARCHAR.
         // https://www.postgresql.org/docs/14/datatype-binary.html
         case "bytea" -> PostgresType.VARCHAR;
+        case "numeric" -> {
+          if (field.get(INTERNAL_DECIMAL_DIGITS) != null && field.get(INTERNAL_DECIMAL_DIGITS).asInt() == 0) {
+            yield BIGINT;
+          } else {
+            yield defaultType;
```

Comment:
shouldn't this be PostgresType.NUMERIC?

Reply:
`numeric` is manually handled in the defaultType. This edge case was added specifically for the no decimals issuel; otherwise, the defaultType will handle the mapping.